### PR TITLE
feat: responsive header, navigation and headings and avatars

### PR DESCRIPTION
**Pull request by pavanjoshi914**
### Describe the changes you have made in this PR

fix headers navigation icons

### Link this PR to an issue [optional]

Fixes _#ISSUE-NUMBER_

### Type of change

- `fix`: Bug fix (non-breaking change which fixes an issue)

### Screenshots of the changes [optional]
before:
image
after:

image

### Checklist

- [X] Self-review of changed code
- [X] Manual testing
- [X] Added automated tests where applicable
- [X] Update Docs & Guides
- For UI-related changes
- [X] Darkmode
- [X] Responsive layout

### CODE REVIEW

**Comment by pavanjoshi914** [comment]
@rolznz @reneaaron here i am thinking what we do about the heading. it's something like ```pavanjoshi2050@gmail.com/Account settings``` this is quite big. and breaks responsiveness on smaller screen. is there specific reason to include account name as well in headings?

**Comment by pavanjoshi914** [comment]
> @rolznz @reneaaron here i am thinking what we do about the heading. it's something like `pavanjoshi2050@gmail.com/Account settings` this is quite big. and breaks responsiveness on smaller screen. is there specific reason to include account name as well in headings?

new changes regarding the same:::

https://github.com/getAlby/lightning-browser-extension/assets/55848322/22041077-ad0e-4bac-b57f-5a9c28aa803b

File: src/app/components/Avatar/index.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ const AvatarImage = (props: Props) => {
       style={{
         width: `${props.size}px`,
         height: `${props.size}px`,
+        flexShrink: 0,
```
Comment (reneaaron) [inline src/app/components/Avatar/index.tsx:29]: Move this to classNames.

File: src/app/components/Avatar/index.tsx [hunk 2]
```diff
@@ -52,7 +53,7 @@ const AvatarSVG = (props: Omit<Props, "url">) => {
   return (
     <svg
       className={classNames(
-        "rounded-full overflow-hidden translate-z-0",
+        "rounded-full overflow-hidden translate-z-0 flex-shrink-0",
```
Comment (reneaaron) [inline src/app/components/Avatar/index.tsx:56]: Isn't that just `shrink-0`? https://tailwindcss.com/docs/flex-shrink

**Review by reneaaron — CHANGES_REQUESTED** [review]

File: src/app/components/Avatar/index.tsx (hunk 1 above)
Comment (pavanjoshi914) [reply src/app/components/Avatar/index.tsx:29]: resolved

**Review by reneaaron — APPROVED** [review]
tACK

**Pull request merged**